src/Icon/Icon.tsx

```diff
@@ -430,6 +430,16 @@ export const Icon: FC<Props> = ({
         />
       </svg>
     )}
+
+    {// POUND
+    render === 'pound' && (
+      <svg width="16" height="16" viewBox="0 0 32 32" fill="none">
```

Comment:
Set the width and height to 100% then it'll be scaleable when imported by other projects.